README.md

```
@@ -25,138 +25,54 @@ models**
-[FiftyOne](https://fiftyone.ai) supercharges your machine learning workflows by
-enabling you to visualize datasets and interpret models faster and more
-effectively.
+![------------------------------------------------------------------](https://github.com/user-attachments/assets/fb0573d0-bb56-40ff-9ae1-a5e8f62f5f42)
 
-Use FiftyOne to get hands-on with your data, including visualizing complex
-labels, evaluating your models, exploring scenarios of interest, identifying
-failure modes, finding annotation mistakes, and much more!
 
-You can get involved by joining our Slack community, reading our blog on
-Medium, and following us on social media:
+## <img src="https://user-images.githubusercontent.com/25985824/106288517-2422e000-6216-11eb-871d-26ad2e7b1e59.png" height="20px"> **installation** 💻
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)